### ISSUE TITLE
why using "was opened" instead of "open"

### ISSUE THREAD

**Issue by migueldc73** [Deejayfool/hass-shutter-card#104]
Superb frontend, I really love it !

I just wonder why you use "was opened" instead of the shorter and more common "open"

This is really an issue for me as I have my covers in 3 rows and the "was opened" and "was closed" text is so long it wraps and shifts the whole button a few pixel down/up when going from percentage to open or closed

**Comment by mochoandre** [comment]
One more for this. For me it does not make sense and uses valuable space. Some versions back it only says open or close.

**Comment by DarKOrange75** [comment]
I don't understand the choice here...

**Comment by Deejayfool** [comment]
If someone find the corresponding labels of "ui.components.logbook.messages.was_closed" and "ui.components.logbook.messages.was_opened", I'll update it with them :) .

**Comment by jluc2808** [comment]
i don't understand what to do , do we have to change something by ourself somewhere ? 
rem:  ui.components.logbook.messages.was_closed  isn't the right label to use because this the old state and the current one

**Comment by Deejayfool** [comment]
No, I can update the card with the good labels, but I don't know where is the list of labels.
It was an other contributor who chose the labels "was_closed" and "was_opened", but I guess the good labels are somewhere else. If someone find them, just let me know and I'll update it.

**Comment by mochoandre** [comment]
It looks like the problem was on Homeassistant side. From some time now it is good and only says open or closed

**Comment by jluc2808** [comment]
@mochoandre it's not ok for me , i always have "était fermé" or "était ouvert" in french and never "fermé" ou "ouvert". 

i have tried to install a fork with "wired label" or the fork of @Criban but for some unknown reason it seems that the old code is still used. 
I have modified the code directly under config/www/community/hass-shutter-card/hass-shutter-card.js , save, restart HA, but still with the old code running, do i have to make another action ?

**Comment by jluc2808** [comment]
as the @Criban fork have some problems (error message :  this is not défined) 
i follow his modification (thanks a lot to @Criban)
i have created a new fork to add optional text: 
- text_opened
- text_closed
- text_no_position (for RTS or other shutter without return position) 

you could find the new fork : 
[https://github.com/jluc2808/hass-shutter-card](url)

the fork could be installed with HACS, while adding personnal repository:  https://github.com/jluc2808/hass-shutter-card
when done with HACS this will overwrite the hass_shutter_card already installed 
you could use without a restart of HA

**Comment by Veldkornet** [comment]
> as the @Criban fork have some problems (error message :  this is not défined) 
> 
> i follow his modification (thanks a lot to @Criban)
> 
> i have created a new fork to add optional text: 
> 
> - text_opened
> 
> - text_closed
> 
> - text_no_position (for RTS or other shutter without return position) 
> 
> 
> 
> you could find the new fork : 
> 
> [https://github.com/jluc2808/hass-shutter-card](url)
> 
> 
> 
> the fork could be installed with HACS, while adding personnal repository:  https://github.com/jluc2808/hass-shutter-card
> 
> when done with HACS this will overwrite the hass_shutter_card already installed 
> 
> you could use without a restart of HA 
> 
> 

Can't you just make a pull request for it to this repository instead of having all these forks?

**Comment by jluc2808** [comment]
yes 

> > as the @Criban fork have some problems (error message :  this is not défined)
> > i follow his modification (thanks a lot to @Criban)
> > i have created a new fork to add optional text:
> > 
> > * text_opened
> > * text_closed
> > * text_no_position (for RTS or other shutter without return position)
> > 
> > you could find the new fork :
> > [https://github.com/jluc2808/hass-shutter-card](url)
> > the fork could be installed with HACS, while adding personnal repository:  https://github.com/jluc2808/hass-shutter-card
> > when done with HACS this will overwrite the hass_shutter_card already installed
> > you could use without a restart of HA
> 
> Can't you just make a pull request for it to this repository instead of having all these forks?

yes but the 1rst text in readme said that the main work isn't maintained and i don't know how i could make a pull request

**Comment by Veldkornet** [comment]
> yes 
> 
> 
> 
> > > as the @Criban fork have some problems (error message :  this is not défined)
> 
> > > i follow his modification (thanks a lot to @Criban)
> 
> > > i have created a new fork to add optional text:
> 
> > > 
> 
> > > * text_opened
> 
> > > * text_closed
> 
> > > * text_no_position (for RTS or other shutter without return position)
> 
> > > 
> 
> > > you could find the new fork :
> 
> > > [https://github.com/jluc2808/hass-shutter-card](url)
> 
> > > the fork could be installed with HACS, while adding personnal repository:  https://github.com/jluc2808/hass-shutter-card
> 
> > > when done with HACS this will overwrite the hass_shutter_card already installed
> 
> > > you could use without a restart of HA
> 
> > 
> 
> > Can't you just make a pull request for it to this repository instead of having all these forks?
> 
> 
> 
> yes but the 1rst text in readme said that the main work isn't maintained and i don't know how i could make a pull request 

Indeed, I missed that. The down side of these HACS integrations...

**Comment by jluc2808** [comment]
Yes, it's also one of several reason i made a fork, this one is maintained :)
is there a problem with my fork ?

**Comment by Veldkornet** [comment]
> Yes, it's also one of several reason i made a fork, this one is maintained :)
> 
> is there a problem with my fork ? 

Well, if you could add this then that would be great! 😬

--> https://github.com/Deejayfool/hass-shutter-card/issues/123
